product_code_builder/models/product_code.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#
+#   Module for OpenERP
+#   Copyright (C) 2015 Akretion (http://www.akretion.com). All Rights Reserved
+#   @author Benoît GUILLOT <benoit.guillot@akretion.com>
+#
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU Affero General Public License as
+#   published by the Free Software Foundation, either version 3 of the
+#   License, or (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+from openerp import fields, models
+
+
+def update_null_and_slash_codes(cr):
```

Comment:
I am confused,  I wrote that for product_sequence.  It shouldn't be here.
Maybe a rebase gone wrong

On Thu, 2 Jul 2015 9:13 PM David Beal <notifications@github.com> wrote:

> In product_code_builder/models/product_code.py
> <https://github.com/OCA/product-attribute/pull/70#discussion_r33759639>:
>
> > +#   License, or (at your option) any later version.
> > +#
> > +#   This program is distributed in the hope that it will be useful,
> > +#   but WITHOUT ANY WARRANTY; without even the implied warranty of
> > +#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
> > +#   GNU Affero General Public License for more details.
> > +#
> > +#   You should have received a copy of the GNU Affero General Public License
> > +#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
> > +#
> > +###############################################################################
> > +
> > +from openerp import fields, models
> > +
> > +
> > +def update_null_and_slash_codes(cr):
>
> useful ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OCA/product-attribute/pull/70/files#r33759639>.
>